main.go

```diff
@@ -202,3 +215,37 @@ func wrapAuthHandler(handler http.HandlerFunc, htpasswdFile string, host string)
 	authenticator := auth.NewBasicAuthenticator(host, secrets)
 	return auth.JustCheck(authenticator, handler)
 }
+
+func writePidFile(c *config.Config, accessLogger cache.Logger) error {
+	// create a "pid" directory under cache directory and clean up inactive pid file
+	pidPath := filepath.Join(c.Dir, "pid")
```

Comment:
If there can be more than one PID, the director should be `pids` no?

Reply:
yeah, it could be multiple files. but I think 'pid' is better than 'pids' as it conforms the convention with 'ac', 'cas'